src/routes/about.svelte

```diff
@@ -120,55 +86,9 @@
     CAC+US.
 </p>
 
-<h2>Our Activities</h2>
-
-<div class="grid">
-    <div class="photo">
-        <img src="client/about/perform.jpg" alt="Our Performance" />
-    </div>
-    <div class="photo--text">
-        <h3>Our Performance</h3>
-        <p>
-            We have been holding our annual concert since 2011, having the main
-            purpose of showcasing angklung and introducing NUS Angklung Ensemble
-            to the NUS community. In our 2015 concert, “The Chronicle”, we
-            performed 13 well-known songs spanning diverse eras to take the
-            audience in a journey through time. As the biggest concert in the
-            history of NUS Angklung Ensemble, “The Chronicle” successfully
-            attracted 150 people, including the Education Attaché of the Embassy
-            of the Republic of Indonesia in Singapore, Mr. Ismunandar, and the
-            manager of NUS Office of Student Affairs, Ms. Lyana Wang Wanzhen.
-        </p>
-    </div>
-    <div class="photo">
-        <img src="client/about/weekly-prac.jpg" alt="Our Weekly Practice" />
-    </div>
-    <div class="photo--text">
-        <h3>Our Weekly Practice</h3>
-        <p>
-            We have our biweekly practice at NUS CAC Clubroom. The collection of
-            songs we play is extensive, ranging from the traditional Indonesian
-            songs such as Manuk Dadali and Ayam Den Lapeh, to the modern tunes
-            such as Pirates of the Carribean and Winter Games. Apart from that,
-            we also organise a musical camp for all members at the end of the
-            year to prepare ourselves for the upcoming performances and
-            simultaneously build solid ties between the members. While the main
-            activities during the camp are practising and reviewing the songs we
-            have learnt, a series of exhilarating welfare events, is also held
-            so that the members has the chance to get to know each other better.
-        </p>
-    </div>
-    <div class="photo">
-        <img src="client/about/workshop.jpg" alt="Our Workshops" />
-    </div>
-    <div class="photo--text">
-        <h3>Our Workshops</h3>
-        <p>
-            We conduct annual workshops for external parties keen on learning
-            more about angklung. If you’re interested in collaborating with us
-            for a workshop, feel free to drop us an
-            <a href="mailto:nus.angklung@gmail.com">email</a>
-            or contact us using the form below!
-        </p>
-    </div>
-</div>
+<h2 style="text-align: center;">Our Activities</h2>
+
+<Carousel items={content} let:blah bind:visibleindex>
+    <CarouselItem {...blah} />
```

Comment:
This is a good of way of using component. Now, I sort of understand why you are separating the content from the html component.

Regardless, this separation will make editing a bit confusing for developers cause they are at different place and carouselItem will be rather limited. Instead of this, there's a better way to improve this.

We can actually extend the functionality of CarouselItem Component by using https://svelte.dev/tutorial/slots (making it more flexible)

With slots, you can put the content directly as a child of CarouselItem.

For instance, something like
<CarouselItem_><p_> Testing </p_></CarouselItem_>

Once you tried it, you will see how the CarouselItem turns into something more versatile.

As a clearer guidance, this is something what we are looking for. 
![image](https://user-images.githubusercontent.com/39835686/110765502-34a38d00-828f-11eb-8061-3cdcb436b37f.png)
It's just that each <div> </div> will be replaced with <CarouselItem> </CarouselItem>